packages/react/src/components/Carousel/Carousel.tsx

```diff
@@ -251,18 +262,33 @@ class Carousel extends AutoControlledComponent<WithAsProp<CarouselProps>, Carous
     }
   }
 
-  renderContent = (accessibility, styles, unhandledProps) => {
-    const { ariaRoleDescription, getItemPositionText, items } = this.props
-    const { activeIndex, itemIds } = this.state
+  overrideItemProps = predefinedProps => ({
```

Comment:
Does this means that we will rerender the whole `Carousel` base on focus event in `CarouselItem`? 💣

Reply:
Well if we could have used `focus-within` we could have avoid it.. Do you have alternative idea?

Reply:
DOM attributes 💣 

Let's go with the current approach now